packages/amazon-sumerian-hosts-babylon/src/Babylon.js/HostObject.js

```diff
@@ -90,14 +92,22 @@ class HostObject extends CoreHostObject {
 
   /**
    * @private
+   * @param {AWS.Polly} polly An AWS Polly service client, assumed to have the proper
```

Comment:
jsdoc has a special notation for indicating optional arguments. Change `{AWS.Polly}` to `{AWS.Polly=}`. The resulting documentation will make it clear that this argument is optional. Do the same for the optional `presigner` argument.